project/npda/migrations/0021_submission_csv_file_name_alter_submission_csv_file.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 5.1.5 on 2025-01-31 17:17
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("npda", "0020_patient_location_bng_patient_location_wgs_and_more"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="submission",
+            name="csv_file_name",
+            field=models.CharField(
+                help_text="Name of the uploaded CSV file",
+                null=True,
+                verbose_name="CSV file name",
+            ),
+        ),
+        migrations.AlterField(
```

Comment:
Suprisingly this is a backwards compatible change, existing rows end up with the old CSV filename as the CSV data. We should clear out staging just to avoid confusion.